examples/todoapp/buildSrc/gradle.properties

```diff
@@ -1,2 +1,3 @@
+# TODO can we get rid of duplication with root gradle.properties?
```

Comment:
Perhaps migrating to "Version catalogs" may help: https://docs.gradle.org/current/userguide/platforms.html

Reply:
But it is not easy to override the version in version catalog via an argument in command line.